util/src/test/java/com/netcracker/ConsoleHelperTest.java

```diff
@@ -0,0 +1,89 @@
+package com.netcracker;
+
+import org.junit.AfterClass;
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.contrib.java.lang.system.TextFromStandardInputStream;
+import org.junit.rules.ExpectedException;
+
+import java.io.*;
+import java.lang.reflect.Constructor;
+import java.lang.reflect.Field;
+import java.lang.reflect.InvocationTargetException;
+import java.lang.reflect.Modifier;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.contrib.java.lang.system.TextFromStandardInputStream.emptyStandardInputStream;
+
+public class ConsoleHelperTest {
+    private static final PrintStream SYSTEM_OUT = System.out;
+    @Rule
+    public final TextFromStandardInputStream systemIn = emptyStandardInputStream();
```

Comment:
That @Rule you use for input stream is a decorator for input stream. It adds behavior to it.
You can and should use analogous @Rule for output stream to decorate it with the ability to read from it after writing to System.out. Take a look at [this](http://stefanbirkner.github.io/system-rules/#SystemErrAndOutRule)